<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="3" column="0" colspan="2" >
      <widget class="QStackedWidget" name="stackedWidget" >
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="page" >
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>8</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="4" column="0" >
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>Program</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" >
          <widget class="QSpinBox" name="program" />
         </item>
         <item row="3" column="1" >
          <widget class="QSpinBox" name="lbank" />
         </item>
         <item row="2" column="1" >
          <widget class="QSpinBox" name="hbank" />
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>L-Bank</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>H-Bank</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2" >
          <widget class="QPushButton" name="patchName" >
           <property name="text" >
            <string>Patch Name</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page" >
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>8</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="2" column="0" colspan="2" >
          <widget class="QLabel" name="label_9" >
           <property name="text" >
            <string>Value</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>Controller</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2" >
          <widget class="QLabel" name="controllerName" >
           <property name="text" >
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="4" >
          <widget class="QSlider" name="valSlider" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="3" >
          <widget class="QSpinBox" name="valSpinBox" />
         </item>
         <item row="0" column="1" colspan="2" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QSpinBox" name="midiPort" />
     </item>
     <item row="0" column="1" >
      <widget class="QSpinBox" name="spinBox" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Time Position:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Midi Port:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QSpinBox" name="midiChannel" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Midi Channel:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Available Controller:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="ctrlList" />
     </item>
     <item>
      <widget class="QPushButton" name="buttonNewController" >
       <property name="text" >
        <string>Create New Controller</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
